v2/tests/results/cover.out

```diff
@@ -0,0 +1,1443 @@
+mode: atomic
+github.com/openshift/oc-mirror/v2/pkg/additional/collector.go:37.103,41.29 2 1
```

Comment:
The commit message doesn't explain what this file is about.  Are you intentionally adding it to version control?  If so, can you update your commit message to give some context for this addition?

Reply:
I am not intentionally adding it. I think this is result of some previous change which picked up my PR after I ran `make all`.

Reply:
@aguidirh, looks like this might be because of [your change][1] in #692?  Can you tell us whether this file is expected to be version-controlled, or if `cover.out` should be added to `.gitignore` or some such?

[1]: https://github.com/openshift/oc-mirror/pull/692/files#diff-76ed074a9305c04054cdebb9e9aad2d818052b07091de1f20cad0bbac34ffb52R72

Reply:
@LalatenduMohanty @wking - Yes the file mentioned should be versisoned controlled , we have  added the cover.* to the .gitignore